module/zfs/dbuf.c

```diff
@@ -227,8 +227,8 @@ typedef struct dbuf_cache {
 dbuf_cache_t dbuf_caches[DB_CACHE_MAX];
 
 /* Size limits for the caches */
-static unsigned long dbuf_cache_max_bytes = ULONG_MAX;
-static unsigned long dbuf_metadata_cache_max_bytes = ULONG_MAX;
+static ZFS_MODULE_ULONG dbuf_cache_max_bytes = ULONG_MAX;
```

Comment:
It is probably a moot point now, but I noticed when working on the patch to supercede this that these values need to be changed from `ULONG_MAX` to `UINT64_MAX`.